constitution/boilerplate.md

```diff
@@ -0,0 +1,686 @@
+
+21. **Execution of documents**
+
+> 1. The CIO shall execute documents either by signature or by affixing its
+>    seal (if it has one)
+>
+> 2. A document is validly executed by signature if it is signed by at least
+>    two of the charity trustees.
+>
+> 3. If the CIO has a seal:
+>
+>    a. It must comply with the provisions of the General Regulations; and
+>    b. The seal must only be used by the authority of the charity trustees or
+>       of a committee of charity trustees duly authorised by the charity
+>       trustees. The charity trustees may determined who shall sign any
+>       document to which the seal is affixed and unless otherwise so
+>       determined it shall be signed by two charity trustees.
+
+22. **Use of electronic communications**
+
+> 1. **General**
+>
+>    The CIO will comply with the requirements of the Communications Provisions
+>    in the General Regulations and in particular:
+>
+>    a. The requirement to provide within 21 days to any member on request a hard
+>       copy of any document or information sent to the member otherwise than in hard
+>       copy form;
+>    b. Any requirements to provide information to the Commission in a
+>       particular form or manner.
+>
+> 2. **To the CIO**
+>
+>    Any member or charity trustee of the CIO may communicate electronically
+>    with the CIO to an address specified by the CIO for the purpose, so long
+>    as the communication is authenticated in a manner which is satisfactory
+>    to the CIO.
+>
+> 3. **By the CIO**
+>
+>    a. Any member or charity trustee of the CIO, by providing the CIO with
+>       their email address or similar, is taken to have agreed to receive
+>       communications from the CIO in electronnic form at that address, unless
+>       the member has indicated to the CIO their unwillingness to receive such
+>       communications in that form;
+>    b. The charity trustees may, subject to compliance with any legal
+>       requirements, by means of publication on its website:
+>       i. Provide the members with the notice referred to in clause 19(2)
+>          (Notice of general meetings); and
+>       ii. Give charity trustees notice of their meetings in accordance with
+>           clause 15(1) (Calling meetings); and
+>       iii. Submit any proposal to the members or charity trustees for
+>            decision by written resolution in accordance with the CIO's
+>            powers under clause 18 (Members' decisions) or 18(4) (Decisions
+>            taken by resolution in writing).
+>    c. The charity trustees must:
+>       i. Take reasonable steps to ensure that members and charity trustees
+>           are promptly notified of the publication of any such notice or
+>           proposal; and
+>       ii. Send any such notice or proposal in hard copy form to any member
+>           or charity trustee who has not consented to receive communications in
+>           electronic form.
+
+23. **Keeping of Registers**
```

Comment:
Careful when batch-accepting (or rejecting!) those; this one also puts the “R” of “registers” lowercase, as there does not seem to be any capitalisation of the titles in this document.

```suggestion
23.) **Keeping of registers**
```